templates/expo-template-tabs/package.json

```diff
@@ -14,19 +13,15 @@
     "preset": "jest-expo"
   },
   "dependencies": {
-    "@expo/vector-icons": "^13.0.0",
```

Comment:
imo we should leave this dependency since we import it directly. not going to block this pr though